pilot/pkg/xds/gen.go

```diff
@@ -128,10 +128,10 @@ func (s *DiscoveryServer) pushXds(con *Connection, push *model.PushContext,
 		if log.DebugEnabled() {
 			// Add additional information to logs when debug mode enabled
 			log.Infof("%s: PUSH%s for node:%s resources:%d size:%s nonce:%v version:%v",
-				v3.GetShortType(w.TypeUrl), req.PushReason(), con.proxy.ID, len(res), util.ByteCount(ResourceSize(res)), resp.Nonce, resp.VersionInfo)
+				v3.GetShortType(w.TypeUrl), req.PushReason(), con.ConID, len(res), util.ByteCount(ResourceSize(res)), resp.Nonce, resp.VersionInfo)
```

Comment:
I am not sure this is not too noisy, is the connection id pretty long?

Reply:
nope, just proxyId + connection number

Reply:
it looks like it repeats proxyid multiple times. I don't think a log would fit on a single line on a large monitor anymore which makes the logs hard to see

Reply:
acked